strapi-app/package.json

```diff
@@ -9,7 +9,10 @@
     "build": "strapi build",
     "strapi": "strapi"
   },
-  "devDependencies": {},
+  "devDependencies": {
```

Comment:
I don't think we need huski in the strapi app since we should not be doing any developent on it ourselves.  What do you think @patricksimonian